src/test/java/edu/cornell/kfs/gl/service/impl/fixture/EmailAdressTestValue.java

```diff
@@ -0,0 +1,24 @@
+package edu.cornell.kfs.gl.service.impl.fixture;
+
+public enum EmailAdressTestValue {
```

Comment:
Both enum name and file name contain a misspelling. Should be EmailAddressTestValue.

Reply:
OK